pkg/printer/formatter.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Authors of Hubble
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package printer
+
+import (
+	"strconv"
+)
+
+// uint64Grouping formats n by grouping digits by 3, separated by a comma
+// (e.g. 1000000 is formatted as 1,000,000).
+func uint64Grouping(n uint64) string {
```

Comment:
dibs on never having to maintain this function

Reply:
Could this use the text/messages lib?

https://play.golang.org/p/BPPSSpNqgH2

Reply:
I'm not too worried about this. It's unlikely that we'll touch this function and it's covered by tests. When the need arises to handle floats, ints etc I think it'll be time to pull in `golang.org/x/text/message`. until then I think this function is good enough.